doc/source/guide/acquiring_data/database.rst

```diff
@@ -175,7 +175,7 @@ syntax are introduced.
     >>> import sunpy.data.sample
     >>> database.add_from_file(sunpy.data.sample.AIA_171_IMAGE)
     >>> len(database)
-    1
+    10
```

Comment:
Why if we add just one file `sunpy.data.sample.AIA_171_IMAGE` would we get `10`? I'm not sure that's right